public/Lab-Files/Lesson03/Challenges/box-positioning.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Positioning Challenge</title>
+    
+<link rel="stylesheet" type="text/css" href="box-positioning.css"
+ />
+</head>
+
+<body>
+
+        
+    <div class="box1">box1</div>
+    <div class="box2">box2</div>
+    <div class="box3">box3</div>
+    <div class="box4">box4</div>
+    <div class="box5">box5</div>
+    <div class="box6">box6</div>
+    
+</body>
```

Comment:
Make sure you close your html tag. Your code looks good for the most part, but I think you should focus on organizing your code better, and not having so many blank lines. Also, the closing bracket on line 8 can be on line 7.